### DISCUSSION TITLE
Schedule notifications

### DISCUSSION THREAD

**Discussion by olup** [tauri-apps/tauri#3149] [Q&A] [answered]
Just wondering, there doesn't seem to be an option to schedule a notification for later in the API. ~I know that web notifications can work with worker threads - would it work in a Tauri app too? Or should Install other rust crates to manage per-system notification scheduling?~
Actually, web workers do not support this. Is there a built-in system?

**Comment by FabianLars** [comment, marked as answer]
The underlying crate we use ([notify-rust](https://github.com/hoodie/notify-rust)) supports this only on mac.
As a workaround this could be added in a way that needs the tauri app running all the time, by "scheduling" it in a tokio task or std thread with `sleep()`. Maybe setTimeout in js could work too, i don't know.

But _i_ don't see a implementation in tauri happening until it's properly implemented upstream, so in the time being this would be DIY with the above mentioned workaround :/
